pkgs/tools/security/pwncat/default.nix

```diff
@@ -5,11 +5,11 @@
 
 buildPythonApplication rec {
   pname = "pwncat";
-  version = "0.1.0";
+  version = "0.1.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0sfdqphs0v3lj3vffda4w05r6sqir7qafa8lmlh0wr921wyiqwag";
+    sha256 = "62e625e9061f037cfca7b7455a4f7db4213c1d1302e73d4c475c63f924f1805f";
   };
 
   # Tests requires to start containers
```

Comment:
description should start with a leading space.

Reply:
@SuperSandro2000 Not sure what you are referring to

Reply:
`description = " TCP/UDP communication suite";` should be `description = "TCP/UDP communication suite";`.